keps/sig-cli/1441-kubectl-debug/README.md

```diff
@@ -401,11 +401,11 @@ Probes and labels are be stripped from Pod copies.
 
 #### Profile: netadmin
 
-| Journey             | Debug Container Behavior                                                   |
-| ------------------- | -------------------------------------------------------------------------- |
-| Node                | sets `NET_ADMIN` and privileged; uses host namespaces                      |
-| Pod Copy            | sets `NET_ADMIN` on debugging container                                    |
-| Ephemeral Container | sets `NET_ADMIN` on ephemeral container                                    |
+| Journey             | Debug Container Behavior                                                              |
+| ------------------- | ------------------------------------------------------------------------------------- |
+| Node                | sets `NET_ADMIN` and `CAP_NET_RAW`; uses host namespaces                              |
```

Comment:
nit: elsewhere in this doc we use Docker's syntax that drops the `CAP_` prefix, could you switch from `CAP_NET_RAW` to `NET_RAW`?

Reply:
@verb 
Oops, it's a mistake!
I've fixed it.